Tests/Resources/Fixtures/config/config2.xml

```diff
@@ -11,9 +11,12 @@
     <error enable-parent-provider="true" enable-sibling-provider="true" />
 
     <sitemap enabled="true">
-        <configuration default-change-frequency="never" name="default" guessers="none" voters="none" loaders="none">
+        <configuration default-change-frequency="never" name="default">
             <template format="xml">test.xml</template>
             <template format="html">test.html</template>
+            <loader>_none</loader>
+            <voter>_none</voter>
+            <guesser>_none</guesser>
```

Comment:
this is more verbose now. @WouterJ what is best practice for the xml schema? should we allow both elements and the attribute that is converted by the configuration class to a list?